### DISCUSSION TITLE
Request: Reassign keyboard shortcuts

### DISCUSSION THREAD

**Discussion by 6amane6** [keiwando/toonsquid#166] [Ideas]
Since there’s already a menu for displaying keyboard shortcuts, it would be great if we could reassign them via that menu too!

I use an 8bitdo mini controller with my ipad for drawing support and it has fixed key mappings. Being able to reassign shortcuts to the controllers’ buttons is a major help to my workflow and I’d love to see it here as well!

Really love the app, btw! It’s amazing how natural the interface is!

**Comment by keiwando** [comment]
Thanks for the feedback! I'm sure this will eventually make its way into one of the updates. 
I would actually love to know how many people here use ToonSquid with a keyboard.

**Reply by Cat-Butler** [reply]
Not usually with a full keyboard, but I do have one of those numeric-keypad-sized Aoikyte Procreate keypads with some shortcuts pre-set.

**Reply by shlcoding** [reply]
I use all my animating apps with keyboard

**Reply by ghost** [reply]
Keyboard and mouse support is essential for my workflow, even on iPad!

**Reply by diegostoliar** [reply]
Sorry, I guess I replied as a new entry:
https://github.com/keiwando/toonsquid/discussions/166#discussioncomment-7746407

**Comment by diegostoliar** [comment]
I usually don't use with a keyboard, I really like to do most of the work on the go and on my lap... BUT
I can see me doing 2 workflows if I delve into a more serious project:

1 - Do a lot of work without a keyboard like I usually do, without requiring a desk.
2 - Some times sit down and get down to the grind, and speed up production using a keypad....

I don't think a full keyboard attracts me, but something with a dial would, like Tourbox Elite or Huion Keypad mini.
especially for flipping frames... Toonsquid 3 finger flipping its not as reliable I found.

**Comment by aluminum1** [comment]
Don't have a keyboard, but will get one soon, since I can see how useful it will be when you're doing serious work and need an efficient workflow.

**Comment by ProductDesignMarco** [comment]
I have been using it with keyboard a lot to move from frame to frame and change tools (I come from after effects so I am pretty used to shortcuts), I would love way more shortcuts and customizable ones as well.
